### PR TITLE
#152 fix: alter RUNITDIR for alpine 3.21

### LINKED ISSUE #152: Alpine 3.21 uses a different installation directory for runit
Currently we declare a `RUNITDIR` environment variable for each linux flavor template that specifies the directory that runit is installed to in that flavor of linux. 

For example, the `apk` (alpine) template sets `RUNITDIR` to `/sbin`.
https://github.com/TugboatQA/images/blob/main/templates/Dockerfile.apk.template#L13

That environment variable is then used in [runit-init.sh](https://github.com/TugboatQA/images/blob/7b8b80a59de93324867662245bbd9a67b26bc3fd/dist/runit-init.sh#L5) to execute the various runit commands.

Until now, each flavor has a consistent location for installation. With the latest Alpine release, however, the installation directory has changed from `/sbin` to `/usr/sbin`.

### PR DESCRIPTION
**Pull request by q0rban**
Resolves #152 

## Testing instructions

- Run `task -- alpine`
- `docker run --detach --rm tugboatqa/alpine:3.21.0`
- Then check the status of that container to see if it's healthy ^^
- When done, stop the container with `docker stop`

### CODE REVIEW

File: templates/Dockerfile.apk.template [hunk 1]
```diff
@@ -14,4 +14,10 @@ ENV RUNITDIR=/sbin
 
 COPY runit-init.sh /runit-init.sh
 
+# Alpine 3.21.* installs runit into a different directory. Test if this is 3.21
+# and then modify the above script appropriately.
+RUN source /etc/os-release && \
+    test "${VERSION_ID#3.21.}" = "$VERSION_ID" || \
```
Comment (jcbain) [inline templates/Dockerfile.apk.template:20]: I know we have no way of knowing what future tags will do but would the idea to be to keep adding to this check any tags that also modify the location?

**Review by jcbain — APPROVED** [review]
Makes sense. Have a small question that should not be blocking

File: templates/Dockerfile.apk.template (hunk 1 above)
Comment (q0rban) [reply templates/Dockerfile.apk.template:20]: Good question… alternately, we could assume future versions will keep this change, and only do this modification for lower versions.

Comment (jcbain) [reply templates/Dockerfile.apk.template:20]: My intuition would be for this change to be maintained in future versions but perhaps we should let it fail again to ensure that is the case and if it is we can change it then.

**Comment by q0rban** [comment]
I prefer the solution in #153

**Pull request closed without merging**